src/main/java/yahav/photomosaic/ImagePixels.java

```diff
@@ -2,106 +2,119 @@
 
 import javafx.geometry.Rectangle2D;
 import javafx.scene.image.Image;
-import javafx.scene.image.ImageView;
-import javafx.scene.image.PixelFormat;
 import javafx.scene.image.PixelReader;
 import javafx.scene.paint.Color;
 
 import java.io.File;
 import java.io.FileInputStream;
-import java.io.IOException;
+import java.io.FileNotFoundException;
 import java.util.ArrayList;
 import java.util.List;
 
-public class ImagePixels{
+public class ImagePixels {
 
     public List<Color> squarePixelColors;
-    public List<Color> allImgClrs;
+    public List<Color> allImageColors;
 
     ArrayList<Rectangle2D> squareInput = new ArrayList<>(); //array of input squares to compare to source images
-    public ImagePixels() throws IOException {
-        FileInputStream input = new FileInputStream("src/main/resources/google.png");
-        Image image = new Image(input);
-        ImageView imageView = new ImageView();
-        squarePixelColors = getAvgClrs(image);
-        allImgClrs = getSrcImgAvgColors();
+
+    public ImagePixels() throws FileNotFoundException {
+        squarePixelColors = getQuadrantColors();
+        //allImgClrs = getSrcImgAvgColors();
     }
 
-    public List<Color> getAvgClrs(Image image) {
+    public List<Color> getQuadrantColors() throws FileNotFoundException {
+        FileInputStream input = new FileInputStream("src/main/resources/google.png");
+        Image image = new Image(input);
         List<Color> squareColors = new ArrayList<>();
         int width = (int) image.getWidth();
         int height = (int) image.getHeight();
-        int squares = 50;
-        int xInc = width / squares;
-        int yInc = height / squares;
+        final int SQUARES = 50;
+        final int quadrantSize = SQUARES * SQUARES;
```

Comment:
This isn't right.
Imagine an image that's 100x100.
How many squares will there be? How many pixels in each square?

Reply:
Nevermind. I think I was momentarily confused as to what SQUARES means.